Server/sprawl_calculator/wup_calculator/data_definitions.py

```diff
@@ -5,3 +5,23 @@
 class WUPCalculationRequestActivating:
 	status: str
 	processing_id: str
+
+@dataclass
+class WUPCalculationRequestProcessing:
+	status: str
+	processing_id: str
+
+@dataclass
+class WUPCalculationRequestCompleted:
+	status: str
+	processing_id: str
```

Comment:
There is a extra result field here, that needs to be added: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L271

Reply:
The results field needs to be updated, see the WUPCalculationResult definition here: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L284

Reply:
Yes, lets create a new dataclass called WUPCalculationResult per definition here: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L202 and then there should be a field here in WUPCalculationRequestCompleted called result of type WUPCalculationResult `result: WUPCalculationResult`

Reply:
The result should be added to the successful processing class